setup.py

```diff
@@ -63,23 +65,23 @@
     description="Simularium Conversion helps convert ssimulation outputs to the format consumed by the Simularium viewer.",
     entry_points={
         "console_scripts": [
-            "my_example=simularium_conversion.bin.my_example:main"
+            "my_example=simulariumio.bin.my_example:main"
```

Comment:
Do you want to name one of the bin scripts "my_example" ?

Reply:
I did the example in jupyter notebook, should I add my_example.py too?